model/work.py

```diff
@@ -874,6 +875,17 @@ def calculate_presentation_edition(self, policy=None):
         )
         return changed
 
+    def _get_default_audience(self):
+        """Return the default audience.
+
+        :return: Default audience
+        :rtype: Optional[str]
+        """
+        for license_pool in self.license_pools:
+            if license_pool.collection.default_audience:
+                return license_pool.collection.default_audience
```

Comment:
Since there can be multiple LicensePools for the same work from different collections, this will choose one at random. I was picturing a solution that went into `Metadata.apply` and added some extra information to the Identifier while we know which data source we're talking about. But that doesn't actually remove the random element; it just creates multiple pieces of (potentially conflicting) evidence which the classifier needs to sort out later.

So I'm OK with this solution overall. I don't think the random element will come up in a real case.